### PR TITLE
Added missing explanation of the "container" term to hello-faq.yml on…

### PR DESCRIPTION
**Pull request by vazome**
… the base of retired hello-how-it-works.md.

## Why

**What’s a container?** – elaborately explained the concept of WH4B container, but now is lost because of retirement.
Let's bring it back in a shorter form and provide information on WH4B removal on a configured device.

## Changes

windows/security/identity-protection/hello-for-business/hello-faq.yml

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-faq.yml [hunk 1]
```diff
@@ -63,7 +63,17 @@ sections:
...
           The statement "PIN is stronger than Password" is not directed at the strength of the entropy used by the PIN. It's about the difference between providing entropy versus continuing the use of a symmetric key (the password). The TPM has anti-hammering features that thwart brute-force PIN attacks (an attacker's continuous attempt to try all combination of PINs). Some organizations may worry about shoulder surfing. For those organizations, rather than increase the complexity of the PIN, implement the [Multifactor Unlock](feature-multifactor-unlock.md) feature.
-          
+
+      - question: What's a container?
+        answer: |  
+          In the context of Windows Hello for Business it is shorthand for a logical grouping of key material or data. Windows 10 or Windows 11 Hello uses a single container that holds user key material for personal accounts, including key material associated with the user's Microsoft account or with other consumer identity providers, and credentials associated with a workplace or school account.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-faq.yml:69]:
```suggestion
          In the context of Windows Hello for Business, it is shorthand for a logical grouping of key material or data. Windows 10 or Windows 11 Hello uses a single container that holds user key material for personal accounts, including key material associated with the user's Microsoft account or with other consumer identity providers, and credentials associated with a workplace or school account.
```

File: windows/security/identity-protection/hello-for-business/hello-faq.yml [hunk 2]
```diff
@@ -63,7 +63,17 @@ sections:
...
+          It's important to keep in mind that there are no physical containers on disk, in the registry, or elsewhere. Containers are logical units used to group related items. The keys, certificates, and credentials Windows Hello stores are protected without the creation of actual containers or folders.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-faq.yml:70]:
```suggestion
          It's important to keep in mind that there are no physical containers on disk, in the registry, or elsewhere. Containers are logical units used to group related items. The keys, certificates, and credentials of Windows Hello stores are protected without the creation of actual containers or folders.
```

File: windows/security/identity-protection/hello-for-business/hello-faq.yml [hunk 3]
```diff
@@ -63,7 +63,17 @@ sections:
...
+          The container actually contains a set of keys, some of which are used to protect other keys. The following image shows an example: the protector key is used to encrypt the authentication key, and the authentication key is used to encrypt the individual keys stored in the container. [Each logical container holds one or more sets of keys.](./images/passport-fig3-logicalcontainer.png)
+
+      - question: How to delete Windows Hello for Business container on a device?
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-faq.yml:73]:
```suggestion
      - question: How do I delete a Windows Hello for Business container on a device?
```

File: windows/security/identity-protection/hello-for-business/hello-faq.yml [hunk 4]
```diff
@@ -63,7 +63,17 @@ sections:
...
+        answer: |  
+          You can effectively disable Windows Hello for Business by launching `certutil.exe -deleteHelloContainer` on the end device under a user account. Reboot is required.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-faq.yml:75]:
```suggestion
          You can effectively disable Windows Hello for Business by launching `certutil.exe -deleteHelloContainer` on the end device under a user account, and then restarting the device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @vazome, kindly commit the suggested changes. Many thanks.

**Comment by vazome** [comment]
@JohanFreelancer9 sure

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/identity-protection/hello-for-business/hello-faq.yml [hunk 5]
```diff
@@ -63,7 +63,17 @@ sections:
...
+          It's important to keep in mind that there are no physical containers on disk, in the registry, or elsewhere. Containers are logical units used to group related items. The keys, certificates, and credentials of Windows Hello stores are protected without the creation of actual containers or folders.
+          The container actually contains a set of keys, some of which are used to protect other keys. The following image shows an example: the protector key is used to encrypt the authentication key, and the authentication key is used to encrypt the individual keys stored in the container. [Each logical container holds one or more sets of keys.](./images/passport-fig3-logicalcontainer.png)
+
+      - question: How do I delete a Windows Hello for Business container on a device?
+        answer: |  
+          You can effectively disable Windows Hello for Business by launching `certutil.exe -deleteHelloContainer` on the end device under a user account, and then restarting the device.
+
```
Comment (paolomatarazzo) [inline windows/security/identity-protection/hello-for-business/hello-faq.yml:76]: can you please update with the following (taken from the retired article):

      - question: What's a container?
        answer: |  
          In the context of Windows Hello for Business, it's shorthand for a logical grouping of key material or data. Windows Hello uses a single container that holds user key material for personal accounts, including key material associated with the user's Microsoft account or with other consumer identity providers, and credentials associated with a workplace or school account.
          The container holds enterprise credentials only on devices that have been registered with an organization; it contains key material for the enterprise IDP, such as on-premises Active Directory or Azure AD.
          Note that there are no physical containers on disk, in the registry, or elsewhere. Containers are logical units used to group related items. The keys, certificates, and credentials of Windows Hello stores, are protected without the creation of actual containers or folders.
          The container contains a set of keys, some of which are used to protect other keys. The following image shows an example: the protector key is used to encrypt the authentication key, and the authentication key is used to encrypt the individual keys stored in the container. [Each logical container holds one or more sets of keys.](./images/passport-fig3-logicalcontainer.png)

**Comment by paolomatarazzo** [comment]
hello @vazome and thank you for the contribution. Can you please update looking at the proposed change?

File: windows/security/identity-protection/hello-for-business/hello-faq.yml (hunk 5 above)
Comment (vazome) [reply windows/security/identity-protection/hello-for-business/hello-faq.yml:77]: Done

**Pull request merged**